### PR TITLE
More Explosion logging

### PR DESCRIPTION
**Pull request by TheOneAndOnlyCreeperJoe**
One step closer to becoming a totalitarian server.

Adds logs to chemistry explosions, pipe bomb explosions, door charges, explosive implants, AI explosions, slime oil explosions and flux anomaly explosions.
Adds logs to people drinking from the glass goblet.
All explosion logs now have a JMP button behind them.

### CODE REVIEW

**Comment by ghost** [comment]
+1 yeah sick cunt fuck yeah

**Comment by ArcLumin** [comment]
does this include plasma in cigar explosions?

File: html/changelogs/Creeper Joe - totalitarian logging.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+author: Creeper Joe
+
+delete-after: True
+
+changes:
+  - tweak: "Becoming a vampire trough the filled glass goblet is now properly logged."
```
Comment (JohnGinnane) [inline html/changelogs/Creeper Joe - totalitarian logging.yml]: "trough" > "through"

**Comment by TheOneAndOnlyCreeperJoe** [comment]
it doesn't include plasma cigars yet, that'll come another time.

**Review by JohnGinnane — CHANGES_REQUESTED** [review]

**Comment by JohnGinnane** [comment]
Did you test all the explosions that you added logging to?

**Comment by TheOneAndOnlyCreeperJoe** [comment]
All of them, yes.

**Comment by ArcLumin** [comment]
"It's a classic case of "I don't know how it works it just works"" - Joe, 2016

code looks good, albeit I have no idea how the syndie bomb works on non-carbon life. IT JUST WERKS, it seems

**Review by JohnGinnane — APPROVED** [review]

**Comment by JohnGinnane** [comment]
Code looks good

**Comment by Swagile** [comment]
yes please our god creeper strikes

**Comment by JohnGinnane** [comment]
Need @Hackole input

**Comment by TheOneAndOnlyCreeperJoe** [comment]
Added logging for oil slime extract explosions too.

**Comment by Swagile** [comment]
GM + 1, good shit

**Comment by SpankMasterson** [comment]
ye

**Comment by TheOneAndOnlyCreeperJoe** [comment]
this should have long been merged

**Pull request merged**